lib/indexer/document_preparer.rb

```diff
@@ -27,7 +38,7 @@ def prepare_popularity_field(doc_hash, popularities)
     end
 
     def prepare_tags_field(doc_hash)
-      Indexer::TagLookup.new.prepare_tags(doc_hash)
```

Comment:
For completeness here - decided to keep the class method interface after all because minitest makes it unreasonably weird to stub out this method call.